pkg/doc/verifiable/test-suite/config.json

```diff
@@ -7,7 +7,7 @@
     "es256kPrivateKeyJwk": {
       "kty": "EC",
       "kid": "did:example:0xab#verikey-1",
-      "crv": "P-256K",
+      "crv": "secp256k1",
```

Comment:
This is a correct curve name according to https://tools.ietf.org/html/draft-ietf-cose-webauthn-algorithms-03